lib/Twig/Extension/Core.php

```diff
@@ -1116,7 +1116,7 @@ function _twig_escape_css_callback($matches)
     // \xHH
     if (!isset($char[1])) {
         $hex = ltrim(strtoupper(bin2hex($char)), '0');
-        if (0 === strlen($hex)) {
+        if ('' === $hex) {
```

Comment:
What makes you think that this is better. I'm sure that the current version is actually faster. Have you some numbers to back this change? In any case, I think the different is really small and that it's just not worth it.